### PR TITLE
Add term Oracle to the Ethereum Glossary

### PR DESCRIPTION
**Pull request by jaskoeth**
## Description

Added the term "Oracle" to the Ethereum Glossary which was previously missing but was directly relevant to Ethereum.
The term was already extensively described here: https://ethereum.org/en/developers/docs/oracles/, but never actually added to the Glossary.

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/4309

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
@jaskoeth Thanks for posting this! Looks good =) 
Just curious thoughts on the relevancy of the last sentence... how often are oracles used for this purpose, as opposed to just querying an RPC connection to the chosen chain directly? Either way, cool with it as is if others are. 👍🏻

**Review by Psyruss77 — APPROVED** [review]

File: src/content/glossary/index.md [hunk 1]
```diff
@@ -473,6 +473,12 @@ A [rollup](#rollups) of transactions that use [fraud proofs](#fraud-proof) to of
 
 <DocLink to="/developers/docs/scaling/layer-2-rollups/#optimistic-rollups" title="Optimistic Rollups" />
 
+### Oracle {#oracle}
+
+An oracle is a bridge between the [blockchain](#blockchain) and the real world. They act as on-chain [APIs](#api) that can be queried for information and used in [smart contracts](#smart-contract). This could be anything from price information to weather reports. Oracles can also be bi-directional, used to "send" data out to the real world.
```
Comment (minimalsm) [inline src/content/glossary/index.md:478]: Agreed @wackerow I also think we could drop the example here as we link out to the same Oracle's doc which has the same example.

```suggestion
An oracle is a bridge between the [blockchain](#blockchain) and the real world. They act as on-chain [APIs](#api) that can be queried for information and used in [smart contracts](#smart-contract).
```

**Comment by jaskoeth** [comment]
Thank you all for reviewing, I agree with your points so I committed the requested change.

**Comment by minimalsm** [comment]
Thanks for the PR @jaskoeth. First of many 🎉

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Pull request merged**